packages/@aws-cdk/aws-apigateway/lib/deployment.ts

```diff
@@ -76,7 +76,7 @@ export class Deployment extends Resource {
     }
 
     this.api = props.api;
-    this.deploymentId = new Token(() => this.resource.deploymentId).toString();
+    this.deploymentId = Lazy.stringValue({ produce: () => this.resource.deploymentId });
```

Comment:
Can we shorten? `Lazy.string` conveys the exact same meaning with less words...

Reply:
`string` will be a keyword  in nearly all languages.